spec/lib/sarif/brakeman_sarif_spec.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../../spec_helper.rb'
+require 'json'
+
+describe Sarif::BrakemanSarif do
+  describe '#parse_issue' do
+    let(:scanner) { Salus::Scanners::Brakeman.new(repository: repo, config: { 'path' => path }) }
+    before { scanner.run }
+
+    context 'scan report with logged vulnerabilites' do
+      let(:repo) { Salus::Repo.new('spec/fixtures') }
+      let(:path) { '/home/spec/fixtures/brakeman/vulnerable_rails_app' }
+      it 'parses information correctly' do
+        brakeman_sarif = Sarif::BrakemanSarif.new(scanner.report)
+        issue = JSON.parse(scanner.log(''))['warnings'][0]
+
+        expect(brakeman_sarif.parse_issue(issue)).to include(
```

Comment:
Be sure to check any produced SARIF against the SARIF schema.  Level here looks suspect.
From the SARIF JSON Schema:
```
        "level": {
          "description": "A value specifying the severity level of the notification.",
          "default": "warning",
          "enum": [ "none", "note", "warning", "error" ]
        },

```

Reply:
the parse_issue function collects the a brakeman result json and creates a hash with information parsed from it. The conversion to the sarif levels is done in the base class.  https://github.com/coinbase/salus/blob/ae0ff1df18f7bea365d5c4afe7fa086bf9ff934f/lib/sarif/base_sarif.rb#L43

this would then convert the parsed severity/level extracted from the brakeman report to a sarif level
```
HIGH ~> error
Medium ~> error
Low ~> warning
Weak ~> note
```